code/Common/SceneCombiner.cpp

```diff
@@ -250,17 +250,18 @@ void SceneCombiner::AttachToGraph(aiScene *master, std::vector<NodeAttachmentInf
 }
 
 // ------------------------------------------------------------------------------------------------
-void SceneCombiner::MergeScenes(aiScene **_dest, std::unique_ptr<aiScene>& master, std::vector<AttachmentInfo> &srcList, unsigned int flags) {
+void SceneCombiner::MergeScenes(aiScene **_dest, aiScene *master, std::vector<AttachmentInfo> &srcList, unsigned int flags) {
+    std::unique_ptr<aiScene> masterAutoPtr(master);
```

Comment:
I would call this `master` and rename the parameter to `master_` or `master_raw`